OpenRA.Mods.Common/Widgets/Logic/SettingsLogic.cs

```diff
@@ -350,18 +350,14 @@ Action InitInputPanel(Widget panel)
 					{ "ToSelectionKey", "Jump to selection" },
 					{ "SelectAllUnitsKey", "Select all units on screen" },
 					{ "SelectUnitsByTypeKey", "Select units by type" },
-
 					{ "PlaceBeaconKey", "Place beacon" },
-
 					{ "PauseKey", "Pause / Unpause" },
 					{ "SellKey", "Sell mode" },
 					{ "PowerDownKey", "Power-down mode" },
 					{ "RepairKey", "Repair mode" },
-
 					{ "NextProductionTabKey", "Next production tab" },
 					{ "PreviousProductionTabKey", "Previous production tab" },
 					{ "CycleProductionBuildingsKey", "Cycle production facilities" },
-
```

Comment:
Please revert this. We added an exception here. https://github.com/OpenRA/OpenRA/pull/7193#discussion_r22280295